### PR TITLE
add support for left-chat-member feature

### PR DESCRIPTION
**Pull request by lov3catch**

### CODE REVIEW

File: src/ChatMember.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 use function Formapro\Values\get_object;
 use function Formapro\Values\get_value;
 
-class NewChatMember
+class ChatMember
```
Comment (makasim) [inline src/ChatMember.php:8]: Why did you rename it? That would be good to keep naming inline with Telegram names.

Comment (lov3catch) [reply src/ChatMember.php:8]: Sorry for pause :(
Rename because it can be NewChatMember or LeftChatMember and has equals fields, so I rename this to more abstraction name.

Comment (makasim) [reply src/ChatMember.php:8]: I'd prefer to have two separate objects and Member interface

Comment (lov3catch) [reply src/ChatMember.php:8]: Ok, I will fix separate this in next PRs.
(But this objects would be identical)

Comment (makasim) [reply src/ChatMember.php:8]: > (But this objects would be identical)

yeap. that's fine
